clang/docs/checkedc/rfc/Include-Checked-Hdrs.md

```diff
@@ -0,0 +1,187 @@
+# Explicit and Implicit Inclusion of Checked Header Files
+Author: Sulekha Kulkarni, Approver: David Tarditi, Date Approved:
+
+
+# OARP
+
+| Role          | Person |
+|-------------------|---------------|
+| Owner         | Sulekha Kulkarni  |
+| Approver      | David Tarditi     |
+| Reviewers     | Joseph Lloyd, Mandeep Grang, Katherine Kjeer |
+| Reviewers     | Mike Hicks and other folks at CCI |
+
+
+## Goal
+The goal of this document is to record the design considerations for allowing
+programs to include the checked counterparts of system header files either
+explicitly or implicitly during the conversion of these programs to Checked C.
+
+## Problem Summary
+The existing approach for including checked header files, which is to explicitly
+replace an included header file with its checked counterpart, is inconvenient
+for automation by 3C. So we need a way to implicitly include checked header
```

Comment:
I would talk about this in more general terms.  3C is representative of an approach to converting files.  It is inconvenient for adoption of Checked C to have to alter include files throughout a project.

Reply:
Thank you for your review! I have addressed all the review comments and made the requisite changes to the doc.